src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,73 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+
+import com.google.codeu.data.User;
+
+
+/**
+ * Responds with a hard-coded message for testing purposes.
+ */
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet{
+
+    private Datastore datastore;
+
+    @Override
+    public void init() {
+        datastore = new Datastore();
+    }
+
+    /**
+     * Responds with the "about me" section for a particular user.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+
+        response.setContentType("text/html");
+
+        String user = request.getParameter("user");
+
```

Comment:
Here I think the line spaces make the code a bit harder to read. I recommend finding digestible, logical chunks of code and separating them with line breaks. Something like:

```java
         response.setContentType("text/html");

         String user = request.getParameter("user");
         if (user == null || user.equals("")) {
            // Request is invalid, return empty response
            return;
         }

         User userData = datastore.getUser(user);
         if (userData == null || userData.getAboutMe() == null) {
            return;
         }
```